### DISCUSSION TITLE
Authelia - The Single Sign-On Multi-Factor portal for web apps

### DISCUSSION THREAD

**Discussion by 391a** [community-scripts/ProxmoxVE#963] [Request script]
### Application Name

Authelia

### Website

https://www.authelia.com/

### Description

Authelia is an open-source authentication and authorization server and portal fulfilling the identity and access management (IAM) role of information security in providing multi-factor authentication and single sign-on (SSO) for your applications via a web portal. It acts as a companion for common reverse proxies.
https://github.com/authelia/authelia

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Aloe-recite** [comment]
Installation is really dead simple, and well explained in the documentation, so much so that even a total noob like me was able to do that easily at first try after creating a Debian CT. A script would be useful of course, but wouldn't add "that" much value IMHO. Moreover, Authelia is proving quite complex to configure, so the real pain comes _after_ the deployment 😄

**Comment by baloo2** [comment]
+1

**Comment by fubarpie** [comment]
would like to see this as well, authentik is more than a lot of people need for their use case and it seems authelia fills that niche

**Comment by thost96** [comment]
Hi all, i've created a PR for that #2060. Please have a look.
